docs/config.json

```diff
@@ -0,0 +1,32 @@
+{
+  "docs": [
+    {
+      "uuid": "7dce5fef-d759-4292-a2b4-ba3ed65f93d7",
+      "slug": "tests",
+      "path": "docs/TESTS.md",
+      "title": "Testing on the Ruby track",
+      "blurb": "Learn how to test your Ruby exercises on Exercism Running"
```

Comment:
```suggestion
      "blurb": "Learn how to test your Ruby exercises on Exercism"
```

Reply:
Or "how to test your solutions to Ruby exercises"? idk :P